doc/releases/changelog-0.11.0.md

```diff
@@ -146,21 +147,20 @@
       return qml.expval(qml.Z(0))
   ```
 
-* Catalyst now supports experimental capture of PennyLane transforms.
+* Catalyst now supports PennyLane transforms captured as plxpr.
```

Comment:
Does this affect the user or is it internal? What is the new thing they get access to ?